galaxy_ng/tests/integration/api/test_namespace_management.py

```diff
@@ -36,7 +36,7 @@ def generate_namespace(exclude=None):
         if exclude is None:
             exclude = []
         namespace = None
-        while namespace is None or namespace in exclude:
+        while namespace is None or namespace in exclude or len(namespace) < 8:
```

Comment:
what is `len(namespace) < 8` for... could it be described in code or comment?

Reply:
8 is an arbitrary number between 2 and 64, chosen to prevent the api from rejecting a namespace of the wrong length:

```
    {
      "status": "400",
      "code": "invalid",
      "title": "Invalid input.",
      "detail": "Name must be longer than 2 characters",
      "source": {
        "parameter": "name"
      }
    }
```

```
    {
      "status": "400",
      "code": "max_length",
      "title": "Invalid input.",
      "detail": "Ensure this field has no more than 64 characters.",
      "source": {
        "parameter": "name"
      }
    }
```